src/RFC-0001_overview.md

```diff
@@ -102,134 +102,112 @@ We can't have fast, cheap digital assets and also highly secure and decentralize
-this we make the explicit trade-off of sacrificing decentralization. Generally, the primary parties that have a stake in the
-security of a given digital asset are the Asset Issuer and Token owners. This fact points to a natural centralization of control
-of an asset by the Asset Issuer.
+### Digital Assets Network
 
-Digital Assets consist of a set of tokens and their associated state. The state of an asset's tokens will be managed on a side-chain
-that will run in parallel to the Tari base layer. The consensus mechanism, ledger style and other characteristics of the side-chain
-will be chosen and managed by the Asset Issuer.
+The DAN is focused on achieving high speed and scalability while maintaining a high degree of decentralisation.
 
-There are many options for the nature of these side-chains. These are still under discussion but it will be possible to run
-multiple types of side-chains in the DAN and an Asset Issuer can choose which best suits the asset type it is supporting.
+The DAN itself is made up of two conceptual levels. On the more fundamental level, the consensus layer uses 
+Cerberus and emergent HotStuff to reach consensus on state changes in the DAN in a highly scalable, decentralised way.
 
-Please refer to Tari Labs University (TLU) for detailed discussions on
-[layer 2 scaling solutions](https://tlu.tarilabs.com/layer2scaling/layer2scaling-landscape/layer2scaling-survey.html)
-and
-[consensus mechanisms](https://tlu.tarilabs.com/consensus-mechanisms/BFT-consensus-mechanisms-applications/Introduction.html).
+A big, and probably the biggest, advantage of this approach is that assets in disparate smart contracts can easily 
+interact, without the need for slow, honey pot-shaped bridges. 
 
-### The role of the base layer
+Then there is the semantic layer, which is where the Tari contracts are compiled, run and verified inside sandboxed 
+Tari virtual machines.
 
-The Base Layer fulfils these, and only these, major roles:
+Together, these levels provide that smart contract enabled digital assets layer, that we've simply been calling the DAN.
 
-1. It manages and enforces the accounting and consensus rules of the base Tari (XTR) token. This includes standard payments,
-   and simple smart contracts such as cross-chain atomic swaps.
-2. It maintains the Digital Asset register. This includes functionality for side-chain management, checkpoints, and transfer
-   of funds into and out of side-chains.
-4. Maintain a register of smart contract templates. This allows users to verify that digital assets are running the code that they 
-   expect and includes functionality like version tracking.
+### Interplay of the layers
 
-In general, the base layer knows _nothing about the specifics of what is happening on the side-chains_. It only cares
+In general, the base layer knows _nothing about the specifics of what is happening on the side-chain_. It only cares
```

Comment:
It does know about Epochs, Validator nodes and templates, but I understand what you're saying

Reply:
And even so, The consensus code only has the vaguest sense. It's metadata attached to std Tari txs. How much validation are we planning on doing on that metadata in the consensus validation code?